### PR TITLE
Feature: Display roles of user

### LINKED ISSUE #31: Display roles of the user
## Description

blocked by #15 
blocked by #12 

On UserPage, add RolesTable with pagination to display all role of the user.

Origin of role and actions are not to implemented in this task.

image

### PR DESCRIPTION
**Pull request by justinehell**
Resolves #31

### CODE REVIEW

File: tests/e2e/features/Users.feature [hunk 1]
```diff
@@ -38,12 +38,18 @@ Feature: Test roundtrip of the application: Users
...
+    ## 201 Should display all users information
     When I click on '[data-cy="users_table"] [data-cy="user_id_1_button_show"]'
     Then I expect current url is '/users/id_1'
     And  I expect '[data-cy="page_user_loading"]' not exists
     And  I expect '[data-cy="page_user_title"]' is 'Firstname'
-    And  I expect '[data-cy="page_user_subtitle"]' exists
+    # Display related roles
+    And  I expect '[data-cy="page_user_roles_title"]' exists
+    And  I expect '[data-cy="roles_table"]' exists
+    And I expect '[data-cy="roles_table"] tbody tr:nth-child(1) td.role-name' is 'admin'
```
Comment (aboubakar14) [inline tests/e2e/features/Users.feature:49]: Not aligned :/

Comment (justinehell) [reply tests/e2e/features/Users.feature]: done

**Review by aboubakar14 — APPROVED** [review]

File: src/components/tables/RolesTable.vue [hunk 2]
```diff
@@ -49,15 +55,16 @@ const columns = ref([{
   field: 'type',
   classes: 'role-type',
 }]);
-const roles = ref([]);
+const roleList = ref([]);
```
Comment (Zorin95670) [inline src/components/tables/RolesTable.vue:58]: useless, you can directly use props in template section.

File: src/components/tables/RolesTable.vue [hunk 3]
```diff
@@ -70,10 +77,10 @@ async function getRolesWithType() {
     }
 
     return role;
-  }));
+  });
 }
 
-onMounted(async () => {
-  roles.value = await getRolesWithType();
+watch(() => props.roles, () => {
```
Comment (Zorin95670) [inline src/components/tables/RolesTable.vue:83]: same

File: src/components/tables/RolesTable.vue (hunk 2 above)
Comment (Zorin95670) [inline src/components/tables/RolesTable.vue:58]: useless you can have use props in template section.

File: src/components/tables/RolesTable.vue [hunk 4]
```diff
@@ -70,10 +77,10 @@ async function getRolesWithType() {
...
+  roleList.value = getRolesWithType(props.roles);
```
Comment (Zorin95670) [inline src/components/tables/RolesTable.vue:84]: not good, have to be done in the parent component.

File: src/components/tables/RolesTable.vue (hunk 2 above)
Comment (justinehell) [reply src/components/tables/RolesTable.vue]: remove

File: src/components/tables/RolesTable.vue (hunk 4 above)
Comment (justinehell) [reply src/components/tables/RolesTable.vue]: done

**Review by Zorin95670 — DISMISSED** [review]

**Review by Zorin95670 — APPROVED** [review]

**Pull request merged**